hs_core/tests/api/native/test_discovery_parser.py

```diff
@@ -77,14 +76,14 @@ def test_parse_with_new_default(self):
     def test_operators(self):
         testcase = {
             "note": str(SQ(content="note")),
-            "need -note": str(SQ(content="need") & ~SQ(content="note")),
-            "need +note": str(SQ(content="need") & SQ(content="note")),
-            "need+note": str(SQ(content="need+note")),
-            "iphone AND NOT subject:10": str(SQ(content="iphone") & ~SQ(
-                subject="10")),
+            # removed '-' 4/5/2019 "need -note": str(SQ(content="need") & ~SQ(content="note")),
```

Comment:
Instead of removing these, could you add a test that tests the new outcome?

Reply:
PyTest!

Reply:
Maybe @zhiyuli , Mahesh or I can look into that or you can do a quick nosetest.